published/external/xdpapi.h

```diff
@@ -51,10 +51,15 @@ XDP_INTERFACE_OPEN_FN(
 
 typedef struct _XDP_API_TABLE XDP_API_TABLE;
 
+
+#define XDP_API_VERSION_1 1
```

Comment:
Do we want to annotate old versions as deprecated (i.e. build warning) to discourage use?

Reply:
Good question - I guess it depends on whether we think this change is worth prodding developers off of. In this case, I don't think it's worth hassling developers.